### PR TITLE
GP-151: Dynamic Fields on ManageCDP page

### PR DESCRIPTION
**Pull request by esyeng**
Changes:
- Patches NaN error for maxGard on MAIN
- Updates logic in WrappedSummary.jsx to cleanly compute both ratio and liquidation price
- Slight formatting adjustments (render modal content top down, add margin to specifics values)

### CODE REVIEW

File: src/components/WrappedSummary.jsx [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+                    marginTop: 20,
+                    marginBottom: 20,
+                    justifyContent: 'space-between',
+                    }}
+                    >
+                    <div>
+                      <SpecificsTitle>{'Transaction Fees'}</SpecificsTitle>
+                    </div>
+                    <div>
+                      <SpecificsValue>{'0.001 Algos'}</SpecificsValue>
```
Comment (mccabe-david) [inline src/components/WrappedSummary.jsx:220]: This value isn't static. It needs to be computed using calcDevFees from cdp.js

File: src/components/WrappedSummary.jsx [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+                      marginTop: 20,
+                      marginBottom: 20,
+                      justifyContent: 'space-between',
+                    }}
+                    >
+                    <div>
+                      <SpecificsTitle>{'New Collateralization Ratio'}</SpecificsTitle>
+                    </div>
+                    <div>
+                      <SpecificsValue>{
```
Comment (mccabe-david) [inline src/components/WrappedSummary.jsx:107]: The position of the elements on the righthand side of the transaction summary are not as they are currently. Please figure out a way to right-align them.

Comment (mccabe-david) [reply src/components/WrappedSummary.jsx:107]: current_alignment
See here for comparison. This happens in both add_collateral and mint

**Review by mccabe-david — CHANGES_REQUESTED** [review]
Looks almost finished! Next time please let me know if you haven't touched the styling!

**Comment by mccabe-david** [comment]
As soon as the values are aligned properly on the right-hand side, we're good to merge!

**Comment by esyeng** [comment]
aligning those now, will pull main once more then should be good to go!

**Comment by esyeng** [comment]
@mccabe-david requesting eyes on this once more :) nothing is broken, things are aligned, dark mode works, the kids are alright

**Comment by mccabe-david** [comment]
> @mccabe-david requesting eyes on this once more :) nothing is broken, things are aligned, dark mode works, the kids are alright (aka the Child component)

Glad the kids are alright lol

**Pull request merged**